source/agora/consensus/protocol/Nominator.d

```diff
@@ -29,15 +29,19 @@ import agora.utils.PrettyPrinter;
 import scpd.Cpp;
 import scpd.scp.SCP;
 import scpd.scp.SCPDriver;
+import scpd.scp.Slot;
 import scpd.scp.Utils;
 import scpd.types.Stellar_types;
 import scpd.types.Stellar_types : StellarHash = Hash;
 import scpd.types.Stellar_SCP;
 import scpd.types.Utils;
 import scpd.Util;
 
+alias TimerType = Slot.timerIDs;
```

Comment:
I  would move it down with other declarations (after the `mixin`) and make it `private`.